Mage/src/main/java/mage/cards/ExpansionSet.java

```diff
@@ -376,7 +374,7 @@ public boolean hasBasicLands() {
         return null;
     }
 
-    public boolean isCustomSet() { return isCustomSet; }
+    public boolean isCustomSet() { return setType == SetType.CUSTOM_SET; }
```

Comment:
Ack, I missed that. My IDE was displaying getters on one line and I mistook that for the project's coding style rather than the IDE doing something weird.